src/main/java/org/sagebionetworks/bridge/dao/ParticipantDataDao.java

```diff
@@ -0,0 +1,58 @@
+package org.sagebionetworks.bridge.dao;
+
+import org.sagebionetworks.bridge.models.ForwardCursorPagedResourceList;
+import org.sagebionetworks.bridge.models.ParticipantData;
+
+public interface ParticipantDataDao {
+
+    /**
+     * Get participant data records for the given userId
+     * @param userId - the userId for the participant data
+     * @return list of participant data records in a resource list that includes original query values
+     */
+    ForwardCursorPagedResourceList<? extends ParticipantData> getParticipantData(String userId, String offsetKey, int pageSize);
+
+    /**
+     * Get participant data for the given userId
+     * @param userId
+     * @param offsetKey
+     * @param pageSize
+     * @return
+     */
+    ForwardCursorPagedResourceList<ParticipantData> getParticipantDataV4(String userId, String offsetKey, int pageSize);
```

Comment:
I don't think you'll need two APIs, one with a "V4" postfix. You are creating the API so there's nothing to support or adapt.